### PR TITLE
bump minor version

### PR DESCRIPTION
**Pull request by sodiumjoe**

### CODE REVIEW

**Comment by sodiumjoe** [comment]
merged master

**Comment by sodiumjoe** [comment]
closed pending some issues in discussion [here](https://github.com/tomaka/winit/pull/465#issuecomment-382146452). Will reopen when that's resolved.

**Comment by sodiumjoe** [comment]
the offending commit was reverted

**Comment by sodiumjoe** [comment]
@francesca64 good question. I'm mostly just trying to follow semver. I know this is pre-1.0, but I still think the minor = features, patch = bugfixes semantics have value. I'd be happy to change it to a patch if that's preferred, though.

**Comment by francesca64** [comment]
My first time trying to write a response, I kept trying to use the term "breaking change", but then I started to reflect on what constitutes a breaking change. For instance, this release contains a bunch of platform parity fixes to the values returned by window position/geometry functions. Someone who only uses winit on one platform wouldn't have had any reason to see the old behavior as a bug, and the behavior of their application could be broken in ways that are more subtly dangerous than breaking API changes.

It's possible that a minor bump would make people more likely to check the changelog, though that could be wishful thinking on my part. On that note, the necessity of checking the changelog is perhaps the least bikesheddy criteria we can come up with for this: a minor release could indicate "we're aware of bad/surprising things that could happen to you if you don't check the changelog" whereas patch releases would be "it's still a good idea to check the changelog, but we're reasonably sure you'll be fine if you don't".

Either way, I definitely want to get #472 resolved before this.

**Comment by sodiumjoe** [comment]
@francesca64 

> Someone who only uses winit on one platform wouldn't have had any reason to see the old behavior as a bug,

I guess it seems to me like one of the main reasons to use winit is to get multi-platform support, so I'm not sure I entirely agree with that. I think I would characterize those issues as consumers relying on buggy behavior.

I also think since winit is still pre-1.0.0, changes like that are especially acceptable.

But I'd be happy to change it to whatever version you think is appropriate.

**Comment by sodiumjoe** [comment]
merged master, which includes the fix for #472

**Comment by francesca64** [comment]
@sodiumjoe not all developers have the luxury of being able to test things on multiple platforms, and our documentation doesn't always do a good job informing you of what's *supposed* to happen. All the shenanigans with `Closed` are a potent example of that in action. That said, I agree with you in cases where the intended behavior is established, it's just that we need more thorough documentation for that to generally be the case.

Anyway, I don't actually have any objections. What's your stance on whether or not #476 should be included? It just needs a macOS review and it should be good to go, though I'd understand if you don't want a breaking API change tacked onto this release.

**Comment by sodiumjoe** [comment]
Makes sense to me to include it.

**Comment by sodiumjoe** [comment]
will reopen once https://github.com/tomaka/winit/pull/476 is merged

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+# Version 0.13.0 (2018-04-17)
+
 - Implement `WindowBuilder::with_maximized`, `Window::set_fullscreen`, `Window::set_maximized` and `Window::set_decorations` for MacOS.
 - Implement `WindowBuilder::with_maximized`, `Window::set_fullscreen`, `Window::set_maximized` and `Window::set_decorations` for Windows.
 - On Windows, `WindowBuilder::with_dimensions` no longer changing monitor display resolution.
```
[the remark this answers is not in the document]
Comment (sodiumjoe) [reply CHANGELOG.md]: done

**Pull request merged**